app/frontend/src/assets/scss/style.scss

```diff
@@ -480,6 +480,25 @@ a,
   }
 }
 
+// allow dropdowns to continue expanding downwards within a modal
+.formio-dialog-content {
+  .component-edit-container {
+    overflow: visible;
+  }
+}
+
+.choices.form-group.formio-choices {
+  .choices__list.choices__list--dropdown {
+    overflow: auto;
+  }
+}
+
+.choices__list.choices__list--dropdown {
+  div {
+    max-height: 175px;
```

Comment:
Is there any way to let this be relative to remaining parent div space instead of a hard-coded pixel count?